index.html

```diff
@@ -13,43 +13,41 @@
     <header>
         <nav>    
             <ul>
-            <div>
                 <li><a href="index.html">Home</a></li>
                 <li><a href="links.html">Link</a></li>
                 <li><a href="glossary.html">Glossary</a></li> 
-             </div>   
```

Comment:
This div is invalid inside the `<ul>`  (only `<li>` elements are allowed as immediate children of a `<ul>`